Gemfile

```diff
@@ -1,6 +1,9 @@
 source 'https://rubygems.org'
-gem 'omnibus', git: 'git://github.com/datadog/omnibus-ruby.git', branch: ENV['OMNIBUS_RUBY_BRANCH']
-gem 'omnibus-software', git: 'git://github.com/datadog/omnibus-software.git', branch: ENV['OMNIBUS_SOFTWARE_BRANCH']
+
+# we default if env variable aren't set or set to an empty string
```

Comment:
This is fine I guess... I do feel it may be a bit too complex for a gemfile. But I guess we don't have a ton of alternatives